README.md

```diff
@@ -65,12 +65,14 @@ ReactDOM.render(
 );
 ```
 
-### Building an embedded app
+### Building an embedded app (deprecated)
```

Comment:
building an embedded app with polaris is definitely not deprecated, probably should communicate this better

Reply:
I think just the point below is enough. We're not preventing people from creating embedded apps. I think this file might be getting some ❤️  in the v5 branch that @BPScott is currently working on. I'd wait for him to chime in on this